desktop/src/app/components/navbar/warnings/modals/convert-field-data-modal.component.ts

```diff
@@ -24,7 +24,7 @@ export class ConvertFieldDataModalComponent {
     public inputTypeLabel: string;
 
     public convertAll: boolean;
-    public countAffected: string;
+    public countAffected: Number;
```

Comment:
For numbers, the TypeScript primitive "number" should be used ("number" instead of "Number").